repro.in

```diff
@@ -377,31 +375,28 @@ function cmd_check(){
 
     # Father I have sinned
     if ((!pkgbuild_file)); then
-    msg2 "Fetching PKGBUILD from ASP..."
+    msg2 "Fetching PKGBUILD from git..."
 
     # Lock the cachedir as we might have a race condition with pacman -S and the cachedir
     lock 9 "${cachedir}.lock"
 
     EPHEMERAL=1 exec_nspawn root --bind="${build_root_dir}/startdir:/startdir" --bind="$(readlink -e ${cachedir}):/var/cache/pacman/pkg" \
     bash <<-__END__
 shopt -s globstar
-pacman -S asp --noconfirm --needed
-if ! asp checkout $pkgbase; then
+
+pacman -S git --noconfirm --needed
+
+if ! git clone --depth 1 --branch "$pkgver" "https://gitlab.archlinux.org/archlinux/packaging/packages/$pkgbase.git"; then
```

Comment:
This won't work as Gitlab restricts some characters in project names or even names such as `tree`. `pkgctl` has a custom lookup function for this. This is why we should use `pkgctl`

Reply:
pkgctl repo clone --switch "${pkgver}" "${pkgbase}"

Reply:
Doesn't work yet:
```
==> ERROR: invalid argument: --unprivileged
```

Reply:
I've put devtools `1:1.0.1-1` into `extra-testing`, please give it a shot and give back feedback if that works as expected or anything I can quickly fix as well.

It should work as follows:
`pkgctl repo clone --protocol https --switch "${pkgver}" "${pkgbase}"`